web/sites/default/config/group.role.bureau_office-admin.yml

```diff
@@ -93,9 +96,26 @@ permissions:
   - 'delete own group_node:service_location content'
   - 'edit group'
   - 'leave group'
+  - 'translate any group_media:audio entity'
```

Comment:
Wouldn't all group types need these permissions set? This looks like only bureau_office was updated.

Reply:
Sorry, those permissions are misleading. I tried another patch that created group-level translation permissions but it didn't offer anything that wasn't already offered by the existing patch I added, so I ended up removing that patch. But it appears that the permissions I created with it were still exported to the config. TLDR: those group permissions don't currently do anything the universal "translate content I can edit" permission allows the user to edit content in any group type.